charts/stable/dokuwiki/values.yaml

```
@@ -0,0 +1,42 @@
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: linuxserver/dokuwiki
+  # -- image tag
+  tag: version-2020-07-29
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See [image docs](https://docs.kanboard.org/en/latest/admin_guide/docker.html#environment-variables)
+# and [application docs](# https://docs.kanboard.org/en/latest/admin_guide/config_file.html) for more details.
+# @default -- See below (only deviations from the default settings are specified)
+env:
+  # Application Configuration
+  # -- Application Timezone
+  TZ: Europe/London
```

We try to default the TZ variable to UTC.

```suggestion
  TZ: UTC
```